app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    command1, command2, command3 = command_parts
```

Comment:
Check the length of command_parts before unpacking and use descriptive variable names